src/balancer/buffered_request_manager.rs

```diff
@@ -11,20 +11,20 @@ use crate::balancer::buffered_request_count_guard::BufferedRequestCountGuard;
 
 pub struct BufferedRequestManager {
     agent_controller_pool: Arc<AgentControllerPool>,
-    buffered_request_timeout: Duration,
+    request_timeout: Duration,
```

Comment:
I know that is kind of "smurf naming convention" (https://blog.codinghorror.com/new-programming-jargon/ -> 21.), but I want to keep elaborate names like `buffered_requests_timeout`. 

That is because it is easy to search through the entire codebase in the editor, and look for `buffered_request_timeout` variable, and I know I will find all the instances of buffered requests timeout. `request_timeout` can mean anything, so it is more ambiguous - it can mean any type of request in any situation without context.

Reply:
It actually applies to just the buffered requests, not all the requests, so the name was correct. It should stay as `buffered_requests_timeout` imo.

> So there is contradiction in documentation.

Yes, we need to fix the docs. The current README is about Paddler v1, we need to move the docs to a separate repo etc; @malzag is working on the new README template currently